packages/react/src/components/ResourceList/__snapshots__/ResourceList.story.storyshot

```diff
@@ -198,14 +198,14 @@ exports[`Storybook Snapshot tests and console checks Storyshots 1 - Watson IoT/R
           Lorem ipsun dolor sit amet, consectetur adipiscing elit. Nunc dui magna, finibus id tortor sed.
         </div>
         <div
-          className="bx--structured-list-td bx--structured-list-content--nowrap ResourceList__CustomActionDiv-sc-27rnbq-2 gHaTLR"
+          className="bx--structured-list-td bx--structured-list-content--nowrap"
           role="presentation"
         >
           <button
             aria-describedby={null}
             aria-pressed={null}
             className="iot--btn bx--btn bx--btn--ghost"
-            data-testid="resource-list-custom-action-row-0"
+            data-testid="resource-list-custom-action-[object Object]"
```

Comment:
the  [object Object] seems wrong

Reply:
It's done automatically. DO you know how to fix it?

Reply:
The snapshot is updated automatically but the problem comes from the fact that at some point we are adding an object instead of the row-id to the string that makes up the data-testid. This might happen on a higher level where the data-testid is a prop called `testId` or `testID`...

Reply:
Fixed